codox/test-sources/codox_test/alter_meta_clj.clj

```diff
@@ -0,0 +1,21 @@
+(ns codox-test.alter-meta-clj
+  (:require [clojure.java.io :as io]))
+
+(defmacro alter-the-meta-data![ns name meta-changes]
```

Comment:
please leave a space between function name and arglist

Reply:
will do, thanks